crates/evm/evm/src/executors/fuzz/mod.rs

```diff
@@ -108,24 +113,37 @@ impl FuzzedExecutor {
                 FuzzOutcome::Case(case) => {
                     let mut data = execution_data.borrow_mut();
                     data.gas_by_case.push((case.case.gas, case.case.stipend));
+
                     if data.first_case.is_none() {
                         data.first_case.replace(case.case);
                     }
+
                     if let Some(call_traces) = case.traces {
                         if data.traces.len() == max_traces_to_collect {
                             data.traces.pop();
                         }
                         data.traces.push(call_traces);
                         data.breakpoints.replace(case.breakpoints);
                     }
+
                     if show_logs {
                         data.logs.extend(case.logs);
                     }
+
+                    // Collect gas snapshots.
+                    for (group, new_snapshots) in case.gas_snapshots.iter() {
+                        data.gas_snapshots
+                            .entry(group.clone())
+                            .or_default()
+                            .extend(new_snapshots.clone());
+                    }
```

Comment:
```suggestion
                    for (group, new_snapshots) in case.gas_snapshots {
                        data.gas_snapshots
                            .entry(group)
                            .or_default()
                            .extend(new_snapshots);
                    }
```

Reply:
looks like this would overwrite snapshots on every fuzz run

wondering if we should instead calculate average value here somehow?

Reply:
decided to remove gas snapshot from fuzz runs currently as it interferes with FORGE_SNAPSHOT_CHECK. In the future we could look at adding an average case as that output is more predictable.